packages/@okta/vuepress-site/docs/reference/api/oidc/index.md

```diff
@@ -271,14 +271,21 @@ The following parameters can be posted as a part of the URL-encoded form values
+| interaction_code        | Required if `grant_type` is `interaction_code`. The value is what was returned from the `/idx` API. The code has a lifetime of 60 seconds.                                                                                                                                                                                         | String |
 | password                | Required if the grant_type is `password`.                                                                                                                                                                                                                                                                                          | String |
 | redirect_uri            | Required if `grant_type` is `authorization_code`. Specifies the callback location where the authorization was sent. This value must match the `redirect_uri` used to generate the original `authorization_code`.                                                                                                                   | String |
 | refresh_token           | Required if `grant_type` is `refresh_token`. The value is a valid refresh token that was returned from this endpoint previously.                                                                                                                                                                                                   | String |
 | scope                   | Required if `password` is the `grant_type`. This is a list of scopes that the client wants to be included in the access token. For the `refresh_token` grant type, these scopes have to be a subset of the scopes used to generate the refresh token in the first place.                                                             | String |
 | username                | Required if the grant_type is `password`.                                                                                                                                                                                                                                                                                          | String |
 
+#### Parameter details
+* `interaction_code` grant type is an Okta extension to [the OpenID specification](http://openid.net/specs/openid-connect-core-1_0.html#Authentication).
+    All other parameters comply with the OpenID Connect specification and their behavior is consistent with the specification.
+
+* To obtain an interaction handle using the `password` grant type, the following parameters are required in addition to those specified above: `code_challenge`, `code_challenge_method`, `redirect_uri`, and `state`. See the `/interact` endpoint for more details.
```

Comment:
where to find `/interact` endpoint with more details?

Reply:
Adding doc for that in the next ticket. The `/interact` endpoint will be documented on the same page, and will update the link here once that's done.